libs/features/pages/home/src/lib/main/home/home-page.component.ts

```diff
@@ -1,8 +1,23 @@
 import { Component } from '@angular/core';
+import { HeroText } from '@elewa-website/models/schema/ui/texts';
 
 @Component({
   selector: 'elewa-website-home',
   templateUrl: './home-page.component.html',
   styleUrls: ['./home-page.component.scss'],
 })
-export class HomePageComponent {}
+export class HomePageComponent {
+
+  // display the text using text-section component
+
+  public heroTexts: HeroText[] = [
+    {
+      title: 'Training at scale',
+      subtitle: 'Human-centered, meaningful and enjoyable',
+    },
+    {
+      title: 'Content development',
+      description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+    }
+  ];
+}
```

Comment:
Data will be provided by the component that reuses this component. No need to submit this.